### ISSUE TITLE
How to use Sequential models within scoring_fn?

### ISSUE THREAD

**Issue by yatharth96** [tensorflow/ranking#220]
Hi,

I have gone through the examples and code, wasn't able to find any readily available support for training Sequential models as part of scoring. 
The transformation functions provided are all aimed at dense features, and I haven't been able to tweak code to train sequential (using LSTMs particularly) models.

The keras equivalent of my code for generating score,looks like:

```
x = Embedding(VOCAB_SIZE, EMBED_DIM)(example_features) # to accept sequential input data and obtain embedding
x = LSTM(500)(x) 
x = Dense(128, activation='relu')(x)
x = Dense(1)(x) # regressor
return x
```

To be precise, how to accept input data (say, ELWC format) and convert input text into sequential features, that can be passed to the above model?  

Not sure, if this is possible to do. Any help is much appreciated.
Thanks

**Comment by dandanmylady** [comment]
TF_ranking is a great project.  it nearly implemented all:  data format and digestion, score_model, ltr_model, loss, metrics.
But for your case, IMO, it's better to leverage the  loss and metrics only.   implement others yourself to full fill your needs.

**Comment by yatharth96** [comment]
Got it, that makes sense. Thanks

**Comment by dandanmylady** [comment]
I reconsidered your case, and in my opinion, the existing framework can do it.   
Basing on the keras example,  there are context features and example features columns.
you can build different 'tower' for different feature columns, in your case, build a LSTM for the feature columns of text. 
you may need to customized your own subclass of UnivariateRankingNetwork
